Formula/neovim.rb

```diff
@@ -25,15 +12,10 @@ class Neovim < Formula
     sha256 "f563c067954a4c5f98ad2b01ee3219e5d1666f7000917474186127122cf5cc76" => :yosemite
   end
 
-  head do
-    url "https://github.com/neovim/neovim.git"
-
-    depends_on "luajit"
-  end
-
   depends_on "cmake" => :build
-  depends_on "lua@5.1" => :build
   depends_on "pkg-config" => :build
+  depends_on "lua@5.1" => :build
```

Comment:
the build-time dependencies should be alphabetized within their category

Reply:
I was considering that it might be legitimate to put `depends_on "lua@5.1" => :build` together with `depends_on "luajit"`, but nevertheless I have reordered them and both of them are now sorted in the alphabetical order within their very own categories. 🤣